modules/bullet/bullet_physics_server.h

```diff
@@ -405,11 +405,11 @@ class BulletPhysicsServer3D : public PhysicsServer3D {
 
 	virtual int get_process_info(ProcessInfo p_info);
 
+	SpaceBullet *get_space(RID p_rid) const;
+	ShapeBullet *get_shape(RID p_rid) const;
 	CollisionObjectBullet *get_collisin_object(RID p_object) const;
 	RigidCollisionObjectBullet *get_rigid_collisin_object(RID p_object) const;
```

Comment:
"collision" is typo'ed in these methods.

Reply:
Fixed